feconf.py

```diff
@@ -426,6 +426,11 @@ def get_empty_ratings():
 OPPIA_PROJECT_ID = 'dev-project-id'
 GOOGLE_APP_ENGINE_REGION = 'us-central1'
 
+# Used by the Admin SDK to connect with the Firebase Authentication emulator.
+# NOTE: this name is NOT a typo, it is the environment variable expected to be
```

Comment:
...is NOT a typo despite the inclusion of the port number, it is...

Reply:
Done.